bokeh/server/tests/test_utils.py

```diff
@@ -26,25 +24,25 @@ def helper():
     return wait_until(helper)
 
 
-def wait_redis_gone(port):
-    def helper():
-        client = redis.Redis(port=port)
-        try:
-            client.ping()
-            return False
-        except redis.ConnectionError:
-            return True
-    return wait_until(helper)
+# def wait_redis_gone(port):
+#     def helper():
+#         client = redis.Redis(port=port)
+#         try:
+#             client.ping()
+#             return False
+#         except redis.ConnectionError:
+#             return True
+#     return wait_until(helper)
```

Comment:
If these are truly unused, remove instead of comment. Does pyflakes complain about unused functions? These seem like they will be useful in the near future, if they are not causing noisy warnings, I'd say just leave them in.